src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+:root {
+  --one-day-box-size: 100px;
+  --gap-between: 1px;
+  --padding-around: 10px;
+  --border-color: #000;
+  --background-color: #eee;
+  --text-color: #000;
+  --font-size: 30px;
+}
+
+$start-day: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$month-lengths: (28, 29, 30, 31);
+$default-length: 31;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  width: 100%;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: var(--font-size);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--gap-between);
+  padding: var(--padding-around);
+  width: calc((100px * 7) + (1px * 6) + (10px * 2));
```

Comment:
It is a good practice to avoid using magic numbers in the code. Instead of hardcoding values like `100px`, `1px`, `10px`, you should use the variables you have defined earlier. This makes the code more maintainable and easier to read.